lib/cli/src/commands/run/wasi.rs

```diff
@@ -215,7 +238,7 @@ impl Wasi {
     /// Helper function for instantiating a module with Wasi imports for the `Run` command.
     pub fn instantiate(
         &self,
-        store: &mut impl AsStoreMut,
+        store: &mut Store,
```

Comment:
The other one keeps things a bit better if you wants to pass a function context instead of a store.
I'd like to keep the `&mut impl AsStoreMut`

Reply:
let me look into this

Reply:
updated